cmd/bpf2go/platform.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import "runtime"
+
+func getTargetArch() string {
+	if runtime.GOOS != "linux" {
+		panic("Not supported platform " + runtime.GOOS)
+	}
+
+	switch runtime.GOARCH {
```

Comment:
Similar comment as above: runtime.GOARCH is the platform we run on. Instead of only looking at `runtime.GOARCH` we should only use it as a fall back if $GOARCH isn't defined in the environment.

My suggestion: add `goarch string` as a parameter and call it like `getTargetArch(os.Getenv("GOARCH"))`.